modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -35,10 +35,14 @@ export type VerificationSettings = {
 /**
  * Result for a individual signature verification
  */
-export type SignatureVerification = {
-  /** Set to the public key that signed for the signature */
-  signedBy: Buffer | undefined;
-};
+export type SignatureVerification =
+  | {
+      /** Set to the public key that signed for the signature */
+      signedBy: Buffer;
+      /** Set to the signature buffer */
+      signature: Buffer;
+    }
+  | { signedBy: undefined; signature: undefined };
```

Comment:
This type seems a little strange to me, feels like either it should be type above or just `undefined` entirely, not an object with a couple of `undefined` properties. And we can just return `undefined` rather than `{ signedBy: undefined, signature: undefined }` where applicable.